server/EmeraldChameleonChat.Services/AutoMapperProfiles/MapperProfiles.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AutoMapper;
+using EmeraldChameleonChat.Services.Model.DTO.Users;
+using EmeraldChameleonChat.Services.Model.Entity.Users;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmeraldChameleonChat.Services.AutoMapperProfiles
```

Comment:
I like the way I have it setup with two different profiles that create explicit & verbose extension methods that make the code more readable. Thoughts?

Reply:
I think this reduces boilerplate substantially and have both mapping in one place to reduces the work needed to update and maintain. 

Most DTOs will have a simple mapper conversion with no logic built into it. 
Map<source, destination>().ReverseMap():
instead of having 
Map<source, destination>();
Map<destination, source>();
in two separate files. 

Thoughts?

Reply:
I get what you are saying, & I am unsure of the best practice.

The bigger thing is adding this new profile to the IDTOExtension/IEntityExtension files/classes so these mappings can be used with the extension methods.

Reply:
I'm not sure about the extensions part will that suffice ? 
ex.AddProfile(new MapperProfiles());